packages/host/app/components/ai-assistant/new-session.gts

```diff
@@ -86,6 +86,7 @@ export default class NewSession extends Component<Signature> {
         padding: var(--boxel-sp-4xs);
         font: 700 var(--boxel-font);
         line-height: 1.5;
+        text-wrap: pretty;
```

Comment:
I didn’t know about this, nice 🤩

Reply:
I'll check the scroll. Yes I just found out about this property and liked it. However it may have caused the panel's right side alignment issue on percy, so I might revert this.